values.yaml

```diff
@@ -84,14 +84,14 @@ mariadb:
   enabled: true
   replication:
     enabled: false
-  db:
+  auth:
     # define external host, when mariadb is disabled
     host: internal
+    rootPassword: "<pwgen -n 64 1>"
```

Comment:
Maybe we also want a line comment before this, to explain why we need a root password *and* a database user password?

The container requires a root password when upgrading, why it is easier with a static one. Else one has to pick the randomly generated one from stdout before upgrading, and still needs to set it.

Reply:
Thanks for the explanation. This is fine, we have set it already before. But 2 years ago, the root password was set over 
rootUser.password instead of auth.rootPassword
https://github.com/bitnami/charts/pull/3849/files#diff-92d5f13581e5b52e0788d31247f4eff88ebe10a6f145821edb142fbaf24431cdL128

Thank you very much for the upgrade, this was desperately needed :)